libnavigation-core/src/main/java/com/mapbox/navigation/core/trip/session/TripSession.kt

```diff
@@ -23,6 +20,7 @@ internal interface TripSession {
 
     fun start()
     fun stop()
+    fun refresh(locationEngineRequest: LocationEngineRequest)
```

Comment:
Is there a use-case for changing the request in runtime? If there's no clear one, I'd rather remove it.

Reply:
Yeah am also fine to remove, we discussed offline reasons to support it such as; customer wants to lower the request to save battery. Which is something you would do at runtime when battery changes.